.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,296 @@
+Chainer Contribution Guide
+==========================
+
+This is a guide for all contributions to Chainer.
+Anyone that wants to register an issue or to send a pull request should read through this document.
```

Comment:
Consider rewording:
The core developers welcome and appreciate any contributions to Chainer. This document describes the various types of contributions and the procedures that we suggest to follow. Anyone that would like to register an issue or to send a pull request should first read through this document.